fs/glob.ts

```diff
@@ -0,0 +1,10 @@
+import { FileInfo } from "deno";
+import { globrex, GlobOptions } from "./globrex.ts";
+
+
+export function glob(
+  glob: string,
+  options: GlobOptions = {}
+): RegExp[] {
```

Comment:
IMO this should be RegExp rather than RegExp[].

Reply:
Yes this give us flexibility to do both regex and glob in the walk pattern.
Fixed it.